src/darwin/Framework/CHIP/MTRCommissioningDelegate.h

```diff
@@ -0,0 +1,123 @@
+/**
+ *    Copyright (c) 2025 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import <Matter/MTRCommissioneeInfo.h>
+#import <Matter/MTRDeviceAttestationDelegate.h> // For MTRDeviceAttestationDeviceInfo
+#import <Matter/MTRMetrics.h>
+#import <Matter/MTRStructsObjc.h>
+
+@class MTRCommissioningOperation; // Can't be imported, since it needs to reference us.
```

Comment:
Unless you're sub-classing or there are typedefs or similar that you need, I would consider forward declaring classes instead of importing the header as the preferred style anyway.

Reply:
OK.  Comment removed.